src/shared/components/palette/money-gbp/read-money-gbp-field.component.spec.ts

```diff
@@ -175,19 +226,19 @@ describe('ReadMoneyGBPFieldComponent', () => {
         .compileComponents();
 
       fixture = TestBed.createComponent(ReadMoneyGbpFieldComponent);
-      component = fixture.componentInstance;
+      component = fixture.componentInstance as ReadMoneyGbpFieldComponent;
 
+      component.registerControl = REGISTER_CONTROL;
       component.amount = undefined;
       component.caseField = CASE_FIELD;
-      component.formGroup = FORM_GROUP;
```

Comment:
Why have you changed this mechanism? It works everywhere else and the `registerControl` method shouldn't be public as it's only ever invoked from within the component, not externally.

Reply:
Honest truth I have changed the mechanism. I just replicated the changes made from the previous PR.

Reply:
Ah! That would explain it! Yes, that got replaced with the mechanism of setting `component.formGroup`, which did away with the need for a mock `registerControl` method and also the need for the _actual_ `registerControl` method being public. Can you undo these bits of changes, please? :)